dts/arm/rpi_pico/rp2040.dtsi

```diff
@@ -53,6 +53,26 @@
 			status = "okay";
 			label = "PINCTRL";
 		};
+
+		uart0: uart@40034000 {
+			compatible = "rpi,pico-uart";
+			reg = <0x40034000 DT_SIZE_K(4)>;
+			clocks = <&peripheral_clk>;
+			interrupts = <20 3>;
+			interrupt-names = "uart0";
+			label = "UART_0";
+			status = "disabled";
+		};
+
+		uart1: uart@40038000 {
+			compatible = "rpi,pico-uart";
+			reg = <0x40038000 DT_SIZE_K(4)>;
+			clocks = <&peripheral_clk>;
+			interrupts = <21 3>;
```

Comment:
question: is priority `3` set for any particular reason?

Reply:
Not really

Reply:
I would suggest using a mechanism similar to the way NRF_DEFAULT_IRQ_PRIORITY is used in the nRF5 dtsi files. This way your users can override more easily while still keeping a sensible default according to maintainer preference.

Reply:
In my branch I have expanded on these definitions a lot, and have kept the prio '3' as-is.
Would it be possible/sensible to use a macro/constant to replace '3', such as "PRIO_3"?
How about replacing the interrupt numbers (13) with the pico-sdk names (IO_IRQ_BANK0)?